models/topics-model.js

```diff
@@ -0,0 +1,7 @@
+const db = require('../db/connection')
+
+function selectTopics(){
+    return db.query(`SELECT * FROM topics;`)
+}
+
+module.exports = selectTopics
```

Comment:
You're probably going to export multiple functions here so would be best to export an object that you can add to later 👍